app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -445,5 +445,9 @@
 
   <!-- OptionAudioLanguage -->
   <dimen name="option_audio_language_padding_start">16dp</dimen>
-  <dimen name="option_audio_language_padding_end">16dp</dimen>
-</resources>
+  <dimen name="option_audio_language_padding_end">16dp</dimen>\
+  
+  <!-- OptionsStoryTextSize -->
+  <dimen name="option_story_text_size_padding_start">16dp</dimen>
+  <dimen name="option_story_text_size_padding_end">16dp</dimen>
+  </resources>
```

Comment:
Please fix the indentation for last line. There should be no space at the start of resrouces.

Reply:
Done